job/models.py

```diff
@@ -1952,20 +1953,6 @@ class ElectricTariffOutputs(BaseModel, models.Model):
         null=True, blank=True,
         help_text="Business as usual year one value of exported energy"
     )
-    year_one_energy_cost_series_per_kwh = ArrayField(
```

Comment:
@Bill-Becker OK to remove these? Doesn't seem like we report these in REopt.jl

Reply:
Oops, saw this after I approved. These energy and demand cost series are pretty nice actually to plot against dispatch and show how/why it's dispatching in different ways, but since we don't have them in the REopt.jl I'd say if we don't report them in REopt.jl, that's fine to remove for now. Maybe create an issue to add these in REopt.jl and then add back to API. The webtool doesn't use these, so it would just be a convenience thing for REopt.jl/API users.

Reply:
got it--added the issue!